src/components/JobseekerProfileForm/index.tsx

```diff
@@ -1,101 +1,136 @@
-import React from 'react'
-import { useDispatch } from 'react-redux'
+import React, { useState, useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
 import { Form, Row, Col } from 'react-bootstrap'
+import { WithContext as ReactTags } from 'react-tag-input'
+import DatePicker, { DayValue } from 'react-modern-calendar-datepicker'
+import 'react-modern-calendar-datepicker/lib/DatePicker.css'
+import BootstrapSwitchButton from 'bootstrap-switch-button-react'
 
 import { updateJobseekerRequest } from '../../redux/actions/jobseeker'
 import HalfCircle from '../../components/HalfCircle'
 import CustomButton from '../../components/CustomButton'
-import ProfileUpload from './profileUpload'
+import { AppState } from '../../redux/types'
+import UploadImage from '../../components/UploadImage'
+
 const JobseekerProfileForm = () => {
-  const [state, setState] = React.useState({
-    id: '',
+  const [tags, setTags] = useState<any[]>([])
+  const [startingAt, setStartingAt] = useState<DayValue>(null)
+  const [isOpenRelocate, setOpenRelocate] = useState(false)
+  const [state, setState] = useState({
     firstName: '',
     lastName: '',
-    phone: '',
+    contact: '',
     seniority: '',
-    degreeName: '',
-    institutionName: '',
+    degree: '',
+    institute: '',
     skills: [],
-    skillLevel: '',
-    duration: '',
+    workExperience: '',
+    image: '',
+    relocate: isOpenRelocate,
+    startingDate: '',
+  })
+
+  const KeyCodes = {
+    comma: 188,
+    enter: 13,
+  }
+
+  const delimiters = [KeyCodes.comma, KeyCodes.enter]
+
+  const skills = useSelector((state: AppState) => state.resources.skills)
+  const suggestions = skills.map(skill => {
+    return {
+      id: String(skill.id),
+      text: skill.name,
+    }
   })
 
-  // Handler for form inputs
-  const handleChange = (e: React.FormEvent<HTMLFormElement>) => {
-    const target = e.target
-    const value = (target as HTMLInputElement).value
-    const name = (target as HTMLInputElement).name
+  useEffect(() => {}, [suggestions, skills])
```

Comment:
This is empty

Reply:
Seems to work fine without it now. Previously it didn't.